client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -160,16 +174,19 @@ export const addComments = ( { dispatch }, { query: { siteId, status } }, { comm
 			status,
 			tree: [],
 		} );
+		dispatch( updateCommentsQuery( siteId, [], { offset, postId, status } ) );
 		return;
 	}
 
+	dispatch( updateCommentsQuery( siteId, comments, { offset, postId, status } ) );
+
 	const byPost = groupBy( comments, ( { post: { ID } } ) => ID );
 
-	forEach( byPost, ( postComments, postId ) =>
+	forEach( byPost, ( postComments, post ) =>
```

Comment:
Any reason for the name change here?

Reply:
I've defined `postId` above, and changing to `post` here seemed an ok fallback, considering the array is called `byPost`. 🙂